scio-test/src/test/scala/com/spotify/scio/coders/CoderTest.scala

```diff
@@ -585,28 +588,28 @@ final class CoderTest extends AnyFlatSpec with Matchers {
     val opts: PipelineOptions = PipelineOptionsFactory.create()
     opts.as(classOf[ScioOptions]).setNullableCoders(true)
 
-    null.asInstanceOf[String] coderShould roundtrip(opts)
-    null.asInstanceOf[jInt] coderShould roundtrip(opts)
-    null.asInstanceOf[jFloat] coderShould roundtrip(opts)
-    null.asInstanceOf[jDouble] coderShould roundtrip(opts)
-    null.asInstanceOf[jLong] coderShould roundtrip(opts)
-    null.asInstanceOf[jShort] coderShould roundtrip(opts)
-    null.asInstanceOf[(String, Top)] coderShould roundtrip(opts)
-    (null, null).asInstanceOf[(String, Top)] coderShould roundtrip(opts)
-    null.asInstanceOf[DummyCC] coderShould roundtrip(opts)
-    DummyCC(null) coderShould roundtrip(opts)
-    null.asInstanceOf[Top] coderShould roundtrip(opts)
-    null.asInstanceOf[Either[String, Int]] coderShould roundtrip(opts)
+    null.asInstanceOf[String] coderShouldWithOpts opts and notFallback()
```

Comment:
just from a semantics perspective, I feel like having to include the `and` makes things a bit confusing--because `coderShouldWithOpts opts` on its own is not making any assertions. Possible to rework to enable syntax like `coderShouldWithOpts(opts) notFallback()`?

Reply:
changed to:
```
null.asInstanceOf[String] coder WithOptions(opts) should notFallback()
```